test-projects/01-basic-app/tests/acceptance/identity-manager-test.js

```diff
@@ -1,10 +1,14 @@
-import { test } from 'qunit';
-import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+import { module, test } from "qunit";
+import { setupApplicationTest } from "ember-qunit";
+import { setupMirage } from "ember-cli-mirage/test-support";
 
-moduleForAcceptance('Acceptance | Identity manager');
+module('Acceptance | Identity manager', function(hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks)
```

Comment:
How was this test working before if they didnt call setupMirage?

Reply:
@cah-briangantzler I'm not sure tbh. I think there was some automatic mirage start with old acceptance tests setup but I didn't bother to find out